src/views/Result.vue

```diff
@@ -510,30 +291,45 @@ position: relative;
   position: absolute;
 }
 
-button:focus {
-  outline: 0;
+.incomplete-habit-wrapper {
```

Comment:
Can actually do away with the entire `.incomplete-habit-wrapper` block for mobile first, it's all unecessary styling.